components/library/src/content/page.rs

```diff
@@ -683,6 +683,23 @@ Hello world
         assert_eq!(page.slug, "hello");
     }
 
+    #[test]
+    fn can_get_date_from_filename_with_spaces() {
+        let config = Config::default();
+        let content = r#"
++++
++++
+Hello world
+<!-- more -->"#
+            .to_string();
+        let res = Page::parse(Path::new("2018-10-08 - hello.md"), &content, &config, &PathBuf::new());
```

Comment:
Didn't we say the space after the `-` was not meant to be there? Eg the slug should be ` hello` depending on the slugification method chosen. You can override the slug strategy of the config to keep everything so we actually check it's working

Reply:
I've added another test that validates that the space is not lost when using a slugification strategy of off. So yes, this depends on the slugification strategy.